src/app/FeedbackForm.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledQuestion = styled.div`
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  color: rebeccapurple;
+`;
+
+const StyledResponse = styled.div`
+  display: flex;
+  margin-bottom: 20px;
+  color: rebeccapurple;
+  font-size: 0.6rem;
+`;
+
+const StyledButton = styled.button`
+  margin-bottom: 5px;
+  padding: 5px;
+  background: rgba(218, 204, 62, 1);
```

Comment:
i think i mentioned that some time ago. when alpha=1 you dont need rgba :)

Reply:
🙄